src/Hl7.Fhir.Core/Rest/TransactionBuilder.cs

```diff
@@ -268,7 +269,7 @@ private string paramValueToString(Parameters.ParameterComponent parameter)
                     case CodeableConcept codeableConcept:
                         return codeableConcept.ToToken();
                     default:
-                        if (ModelInfoNEW.IsPrimitive(parameter.Value.GetType()))
+                        if (ModelInfoExtensions.IsPrimitive(parameter.Value.GetType()))
```

Comment:
I could not find the class file for ModelInfoExtensions, but I guess we need to assume here al FHIR *primitives* are known to common, right?

Reply:
This class lives in the current Common. And all primitives are in Common.